roles/kube_metrics_server/vars/main.yml

```diff
@@ -2,6 +2,9 @@
 kube_metrics_server__default_values:
   args:
   # Default:
-  #- --source=kubernetes.summary_api:''
-  # For https://github.com/scality/metalk8s/issues/36:
-  - "--source=kubernetes.summary_api:''?useServiceAccount=true&kubeletHttps=true&kubeletPort=10250&insecure=true"
+  - --logtostderr
+  # MetalK8s-specific:
+  - --kubelet-insecure-tls
+  # Required to make sure node hostnames aren't resolved through DNS
```

Comment:
Should we adapt the inventory to not change the hostname of the server and get DNS working ?

Reply:
Then you rely on an external DNS server to resolve the hostname, which is not necessarily the case. Defaulting to InternalIP should be fine in most deployments, no? In the end, that's the IP `kubelet`s are reporting themselves.

Reply:
Yes, indeed. Now wondering why it's not the default

Reply:
maybe for tls...

Reply:
Kubespray not generating certificates correctly is why we need `--kubelet-insecure-tls` in the first place :-(